.travis.yml

```diff
@@ -43,7 +44,7 @@ jobs:
   - stage: compile
     os: osx
     compiler: gcc
-    env: CMAKE_BUILD_TYPE=Debug GTSAM_BUILD_UNSTABLE=OFF
+    env: CMAKE_BUILD_TYPE=Debug GTSAM_BUILD_UNSTABLE=OFF GTSAM_WITH_TBB=OFF
```

Comment:
Why is TBB off for only debug and not for release versions? What is the rationale behind that?

Wouldn't it be better to have a separate stage which compiles with TBB for both Debug and Release and runs the test, in the interest of time?

Reply:
Ignore this. I just saw Frank's comment.